<launch>
	<arg name="frame_id" default="TanwayTP" />
	<node pkg="tanwaylidar_view" name="TanwayLidar_MainNode" type="TanwayLidar_MainNode" output = "screen">
		<param name="frame_id" value="$(arg frame_id)" />
		<param name="LidarType" type="int" value="7"/> 
		
		<!--本机IP地址-->
		<param name="LocalHost" type="string" value="192.168.111.204" />
		<!--本机接收雷达数据的端口号-->
		<param name="LocalPointloudPort" type="int" value="5600" />
		<!--l雷达设备的IP地址-->
		<param name="LidarHost" type="string" value="192.168.111.51" />
		<!--雷达设备发送端的端口号-->
		<param name="LidarPort" type="int" value="5050" />
		<!--组修正角度值1-->
		<param name="CorrectedAngle1" type="double" value="0.0" />		
		<!--组修正角度值2-->
		<param name="CorrectedAngle2" type="double" value="0.1" />	
		<!--组修正角度值3-->
		<param name="CorrectedAngle3" type="double" value="0.2" />	

		<!--滤除相关配置参数-->
		<!--X 安装角度误差，单位：度-->
		<param name="InstallRotateX" type="double" value="0.0" />	
		<!--Y 安装角度误差，单位：度-->
		<param name="InstallRotateY" type="double" value="0.0" />	
		<!--Z 安装角度误差，单位：度-->
		<param name="InstallRotateZ" type="double" value="0.0" />	
		<!--安装高度(距离地平面)，单位：米-->
		<param name="InstallMoveZ" type="double" value="0.6" />	

	</node>

	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find tanwaylidar_view)/rviz/tanwayrun.rviz" />
</launch>
